core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -18,41 +20,40 @@
  */
 
 public class Trigger {
-    public static class TriggerInfo {
-        public String name;
-        public String query;
-        public Map<String,Object> selector;
-        public Map<String, Object> params;
-        public boolean installed;
-        public boolean paused;
-
-        public TriggerInfo(String name, String query, Map<String, Object> selector, boolean installed, boolean paused) {
-            this.name = name;
-            this.query = query;
-            this.selector = selector;
-            this.installed = installed;
-            this.paused = paused;
-        }
+    // public for testing purpose
+    public static final String DB_NON_WRITER_ERROR = """
+            This instance is not allowed to write to the database because it is a follower.
+            Please use these procedures against the database leader.
+            """;
```

Comment:
```suggestion
    public static final String DB_NON_WRITER_ERROR = """
            This instance is not allowed to write to the database because it is a follower.
            Please use this procedure against the database leader.
            """;
```